pom.xml

```diff
@@ -7,16 +7,17 @@
   <parent>
     <groupId>com.hotels</groupId>
     <artifactId>hotels-oss-parent</artifactId>
-    <version>2.0.6</version>
+    <version>4.0.1</version>
   </parent>
 
   <artifactId>quibble</artifactId>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>2.0.0-SNAPSHOT</version>
```

Comment:
We're effectively making a non-backwards compatible change by removing some driver jars so I've bumped the major version.